build/yamls/windows/containerd-with-ovs/containerd-with-ovs.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: antrea-agent
+  name: antrea-agent-windows
+spec:
+  selector:
+    matchLabels:
+      component: antrea-agent
+  template:
+    metadata:
+      annotations:
+        "microsoft.com/hostprocess-inherit-user": "true"
+      labels:
+        component: antrea-agent
+    spec:
+      containers:
+        - name: antrea-agent
+          lifecycle:
+            preStop:
+              exec:
+                command: ["powershell", "-file", "$env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent-Containerd.ps1"]
```

Comment:
Better to be consistent with postStart by adding parameters "-VMSwitchExtension", "disable"

Reply:
Reminder